test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -2247,6 +2247,47 @@ BOOST_AUTO_TEST_CASE(dev_return_name_no_description)
 	checkNatspec(sourceCode, "B", natspec2, false);
 }
 
+BOOST_AUTO_TEST_CASE(dev_different_amount_return_parameters)
+{
+	char const *sourceCode = R"(
+		interface IThing {
+			/// @return x a number
+			/// @return y another number
+			function value() external view returns (uint128 x, uint128 y);
+		}
+
+		contract Thing is IThing {
+			struct Value {
+				uint128 x;
+				uint128 y;
+			}
+
+			Value public override value;
+		}
+	)";
+
+	char const *natspec = R"ABCDEF({
+		"methods":
+		{
+			"value()":
+			{
+			  "returns":
+			  {
+				"x": "a number",
+				"y": "another number"
+			  }
+			}
+		}
+	})ABCDEF";
+
+	char const *natspec2 = R"ABCDEF({
+		"methods": { }
```

Comment:
Can we find a solution where this is not empty? Note that if you use `FunctionType(variableDeclaration)`, you will get a function type that has the correct amount of parameters and return variables.

Reply:
Sure, the ABI function type is the same, else it couldn't override.. that's not the problem. But how would you want to map the two documented parameters to the single struct parameter for this function? How would this inherited documentation look like?